client/web/app/app.ts

```diff
@@ -189,6 +170,10 @@ export class App {
       'SetLanguageRequested',
       this.setAppLanguage.bind(this)
     );
+    this.rootEl.addEventListener(
```

Comment:
Can you move this code up to the original place? So GitHub won't show this as changed code.